themes/developer/developer.theme.sh

```diff
@@ -0,0 +1,229 @@
+#! bash oh-my-bash.module
+
+SCM_THEME_PROMPT_DIRTY=" ${_omb_prompt_brown}✗"
+SCM_THEME_PROMPT_CLEAN=" ${_omb_prompt_bold_green}✓"
+SCM_THEME_PROMPT_PREFIX=" ${_omb_prompt_green}|"
+SCM_THEME_PROMPT_SUFFIX="${_omb_prompt_green}|"
+
+GIT_THEME_PROMPT_DIRTY=" ${_omb_prompt_brown}✗"
+GIT_THEME_PROMPT_CLEAN=" ${_omb_prompt_bold_green}✓"
+GIT_THEME_PROMPT_PREFIX=" ${_omb_prompt_green}|"
+GIT_THEME_PROMPT_SUFFIX="${_omb_prompt_green}|"
+
+RVM_THEME_PROMPT_PREFIX="|"
+RVM_THEME_PROMPT_SUFFIX="|"
+
+function return_delimited {
+	# 1 = flag 2 = tag 3 = value
+	if [[ $1 = "true" ]]; then
+		printf "$2 $3---"
+	elif [[ $1 = "false" ]]; then
+		printf "$2 $3"
+	else
+		printf "$1 $2"
+	fi
+}
+
+function extract_key {
+	value=$(echo "$1" | grep -oP "(?<=$2)[^---]+")
+	printf "$value"
+}
+
+function __bobby_clock {
+	printf "$(clock_prompt) "
+
+	if [ "${THEME_SHOW_CLOCK_CHAR}" == "true" ]; then
+		printf "$(clock_char) "
+	fi
+}
+
+function node_version {
+	val_node=$(node --version)
+	if command -v nvm &>/dev/null; then
+		return_delimited $1 "node" "nvm ${val_node}"
+	else
+		# Si nvm no está instalado, utilizar "njs"
+		return_delimited $1 "node" "njs ${val_node}"
+	fi
+}
+
+function go_version {
+	local val_go=$(go version | cut -d ' ' -f 3 | cut -d 'o' -f 2)
+	return_delimited $1 "go" "go ${val_go}"
+}
+
+function ruby_version {
+	local val_rb=$(ruby --version | cut -d ' ' -f 2)
+	if command -v rvm &>/dev/null; then
+		return_delimited $1 "ruby" "rvm ${val_rb}"
+	else
+		# Si nvm no está instalado, utilizar "njs"
+		return_delimited $1 "ruby" "rb ${val_rb}"
+	fi
+}
+
+function py_version {
+	local val_py=$(python --version | cut -d ' ' -f 2)
+	if command -v conda &>/dev/null; then
+		local condav=$(conda env list | grep '*' | awk '{print $1}')
+		return_delimited $1 "python" "conda<${condav}> ${val_py}"
+	else
+		# Si nvm no está instalado, utilizar "njs"
+		return_delimited $1 "python" "py ${val_py}"
+	fi
+}
+
+function OPi5p_Temp {
+	local temp_opi5p=$(cat /sys/class/thermal/thermal_zone4/temp &)
```

Comment:
What is the intent of `&`? There are many cases even for just `echo`. The command substitution anyway blocks until the stream is closed, so putting `&` doesn't change anything in my understanding.

Reply:
I just removed those meaningless backgrounding. As explained above, the command substitution `$()` blocks until the background process terminates, so it doesn't have the desired effect. We might achieve the original purpose by using temporary files, but it complicates the code excessively. Since these didn't work, we can just drop them.

cc6068b